### PR TITLE
Upgrade dom0

### PR DESCRIPTION
**Pull request by hellais**
This is related to ensuring all our infrastructure is redeployable. 

In this PR I am going to address the issues of upgrading dom0-bootstrap to work with Debian 10 and also include fixes for: https://github.com/ooni/sysadmin/issues/289.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ language: python
 python:
   - "2.7"
 install:
-  - pip install ansible==2.4.2
+  - pip install ansible==2.8.3
```
Comment (bassosimone) [inline .travis.yml]: I see in the README.md that we're forcing `dnspython==1.15.0`. Do we want to do that also here?

File: ansible/roles/dom0-defaults/tasks/main.yml [hunk 2]
```diff
@@ -13,24 +13,22 @@
 
 - name: install some common packages
   apt:
-    name: "{{ item }}"
+    name:
+      - ca-certificates
+      - tmux
+      - screen
```
Comment (bassosimone) [inline ansible/roles/dom0-defaults/tasks/main.yml]: @hellais do you still prefer `screen` over `tmux`?

File: .travis.yml (hunk 1 above)
Comment (hellais) [reply .travis.yml]: That is probably a good idea too.

File: ansible/roles/passwd/tasks/main.yml [hunk 3]
```diff
@@ -1,33 +0,0 @@
-# different system may have different places for bash and nologin
```
[the remark this answers is not in the document]
Comment (bassosimone) [reply ansible/roles/passwd/tasks/main.yml]: OK, thanks for explaining. I believe this is a good idea.

**Pull request merged**